build/tasks/arm-test-deploy.js

```diff
@@ -36,41 +23,55 @@ var bootstrapTest = (artifactsBaseUrl) =>
   params.securityAdminPassword.value = config.deployments.securityPassword;
   params.securityLogstashPassword.value = config.deployments.securityPassword;
 
+  // Some parameters are longer than the max allowed characters for cmd on Windows.
+  // Persist to file and pass the file path for parameters
+  var resourceGroup = "test-" + hostname.toLowerCase() + "-" + t.replace(".json", "") + dateFormat(new Date(), "-yyyymmdd-HHMMssl").replace("+","-")
+  var parametersFile = path.resolve(logDistTmp + "/" + resourceGroup + ".json");
+  fs.writeFileSync(parametersFile, JSON.stringify(params, null, 2));
```

Comment:
Not seeing where we clean up `params` of the long values? Should we take a whitelist approach here for what we write pass as `paramsFile` and scrub that from `params` ? Not a fan of persisting e.g passwords. 

We should also clean this file somewhere after the deploy.

Reply:
Can do something nicer here by passing sensitive parameters on command line and only writing non-sensitive parameters to file. Azure CLI 2.0 allows this :smile: